drainer/config.go

```diff
@@ -109,6 +110,7 @@ func NewConfig() *Config {
 		fmt.Fprintln(os.Stderr, "Usage of drainer:")
 		fs.PrintDefaults()
 	}
+	fs.StringVar(&cfg.NodeID, "node-id", "", "the ID of drainer node; if not specify, we will generate one from hostname and the listening port")
```

Comment:
```suggestion
	fs.StringVar(&cfg.NodeID, "node-id", "", "the ID of drainer node; if not specified, we will generate one from hostname and the listening port")
```

Reply:
Updated. The config.go of pump is also updated. @suzaku